tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -71,6 +86,20 @@ public static IEnumerable<object[]> GetVerifyImagesData()
                 .Select(imageDescriptor => new object[] { imageDescriptor });
         }
 
+        public static IEnumerable<object[]> GetVerifyWCFImagesData()
```

Comment:
This method is just a copy of `GetVerifyImagesData` but referencing `WCFTestData` instead of `TestData`.  Please refactor so that the implementation is shared.  Something like this:

```
public static IEnumerable<object[]> GetVerifyRuntimeImagesData()
{
    return GetVerifyImagesData(TestData);
}

public static IEnumerable<object[]> GetVerifyWcfImagesData()
{
    return GetVerifyImagesData(WcfTestData);
}

private static IEnumerable<object[]> GetVerifyImagesData(IEnumerable<ImageDescriptor> imageDescriptors)
{
    string versionFilterPattern = VersionFilter != null ? GetFilterRegexPattern(VersionFilter) : null;
    string osFilterPattern = OSFilter != null ? GetFilterRegexPattern(OSFilter) : null;
    // Filter out test data that does not match the active os and version filters.
    return imageDescriptors
        .Where(imageDescriptor => OSFilter == null
            || Regex.IsMatch(imageDescriptor.OsVariant, osFilterPattern, RegexOptions.IgnoreCase))
        .Where(imageDescriptor => VersionFilter == null
            || Regex.IsMatch(imageDescriptor.RuntimeVersion, versionFilterPattern, RegexOptions.IgnoreCase))
        .Select(imageDescriptor => new object[] { imageDescriptor });
}
```

Reply:
Thanks for the feedback!  I was about to refactor this part today.